client/src/kentpaglomotan.tsx

```diff
@@ -0,0 +1,34 @@
+import React {Component}from 'react';
+
+class TextFielddAndButton extends Component{
+  contructor(props){
+    super{props}
+    this.state = {fullName: ''}
+  }
+  handleSubmitButton = (event) => {
+    event.preventDefault()
+    const data = this.state
+    console.log("The name submitted is", data)
```

Comment:
Change the console log statement to an alert statement, and make sure you're displaying data.firstName